test/conformance/memory/memory_adapter_level_zero.match

```diff
@@ -0,0 +1,14 @@
+{{OPT}}urMemBufferMultiQueueMemBufferTest.WriteBack
+urMemBufferPartitionWithFlagsTest.Success/*__UR_MEM_FLAG_WRITE_ONLY
+urMemBufferPartitionWithFlagsTest.Success/*__UR_MEM_FLAG_READ_ONLY
+urMemBufferPartitionTest.InvalidValueCreateType/*
+urMemBufferPartitionTest.InvalidValueBufferCreateInfoOutOfBounds/*
+{{OPT}}urMemImageCreateWithNativeHandleTest.Success/*
+{{OPT}}urMemGetInfoImageTest.SuccessSize/*
+{{OPT}}urMemImageCreateTestWithImageFormatParam.Success/*__UR_IMAGE_CHANNEL_ORDER_RGBA__*
+
+# These tests fail in the "Multi device testing" job, but pass in the hardware specific test
+{{OPT}}urMemImageCreateTest.InvalidImageDescStype/*
+{{OPT}}urMemImageCreateTest.InvalidSize/*
+{{OPT}}urMemImageCreateWithHostPtrFlagsTest.Success/*
+{{OPT}}urMemImageGetInfoTest.InvalidSizeSmall/*
```

Comment:
We don't use match files in the CTS anymore, this should be deleted and the tests marked with `UUR_KNOWN_FAILURE_ON()` if they haven't already.